mrmustard/lab_dev/transformations/base.py

```diff
@@ -1,4 +1,4 @@
-# Copyright 2023 Xanadu Quantum Technologies Inc.
+# Copyright 2024 Xanadu Quantum Technologies Inc.
```

Comment:
I think we should leave the original year, otherwise copyright is covered only from 2024